backend/Controllers/MovieController.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using CinemaApp.Models;
+
+namespace CinemaApp.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class MovieController : ControllerBase
+    {
+        private readonly MovieContext _context;
+
+        public MovieController(MovieContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Movie
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Movie>>> Getmovies()
+        {
+            return await _context.movies.ToListAsync();
+        }
+
+        // GET: api/Movie/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Movie>> GetMovie(int id)
+        {
+            var movie = await _context.movies.FindAsync(id);
+
+            if (movie == null)
+            {
+                return NotFound();
+            }
+
+            return movie;
+        }
+
+        
+        [HttpPut("{id}")]
+        public async Task<IActionResult> PutMovie(int id, Movie movie)
```

Comment:
I had the same issue with a previous PR: why are you using db models directly and not DTOs? If you are using models, I can send you whatever data, without any restriction. Why should I care about movie.ID?

Reply:
I'm gonna fix that too.